cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -288,19 +260,6 @@ class fixed_point {
    */
   CUDF_HOST_DEVICE inline fixed_point() : _scale{scale_type{0}} {}
 
-  /**
-   * @brief Explicit conversion operator for casting to floating point types
-   *
-   * @tparam U The floating point type that is being explicitly converted to
-   * @return The `fixed_point` number in base 10 (aka human readable format)
-   */
-  template <typename U,
-            typename cuda::std::enable_if_t<cuda::std::is_floating_point_v<U>>* = nullptr>
-  explicit constexpr operator U() const
```

Comment:
Likewise this conversion from fixed to floating point was already explicit. Implicit conversion to floating point was already impossible. I don't see how forcing a call to `convert_fixed_to_floating()` is any more explicit -- it's just more verbose.